sdcm/nemesis.py

```diff
@@ -105,14 +122,17 @@ def run(self, interval=30):
         self.interval = interval
         while True:
             self._set_current_disruption()
+
             self.disrupt()
             if self.termination_event is not None:
                 if self.termination_event.isSet():
                     self.termination_event = None
                     self.log.info('Asked to stop running nemesis')
                     break
+            self.target_node.running_nemesis = None
```

Comment:
if target node was terminated, for example by terminate and replace nemesis? what will happen here?

Reply:
WHen node is terminated and replaced, the new node with running_nemesis=None properties will be added.